app/views/concerts/_form.html.haml

```diff
@@ -15,7 +15,7 @@
           onclick: "$('.song-counts #new_song').hide(); return false;")
 
 = entry_form(buttons_top: false) do |f|
-  #song-counts-form
+  #song-counts-form{data: {focus: params[:focus]}}
```

Comment:
Idea was to pass a optional focus paramatere, that then directly jumps into focus for the right field, but I had trouble getting it to work with turbolinks.

Reply:
Realized it was stupid to try to do this with javascript... now I set the autofocus in the form, if the object id matched the focus parameter.